pages/artist/_itemId/index.vue

```diff
@@ -1,106 +1,115 @@
-                          <!-- eslint-disable-next-line vue/no-v-html -->
-                          <p class="item-overview" v-html="overview" />
-                        </v-col>
+                  </v-row>
+                  <v-row v-for="appearance in appearances" :key="appearance.Id">
+                    <v-col cols="12">
+                      <div class="d-flex flex-column">
+                        <v-row>
+                          <div
+                            style="width: 125px"
+                            class="ma-2 d-none d-md-block"
+                          >
+                            <card :item="appearance" no-text no-margin />
+                          </div>
+                          <div class="py-2">
+                            <div
+                              class="text-subtitle-1 text--secondary font-weight-medium"
+                            >
+                              {{ appearance.ProductionYear }}
+                            </div>
+                            <nuxt-link
+                              class="link font-weight-bold text-h6 text-md-h4"
+                              tag="h2"
+                              :to="`/item/${appearance.Id}/`"
+                            >
+                              {{ appearance.Name }}
+                            </nuxt-link>
+                          </div>
+                        </v-row>
+                        <v-row class="my-2">
+                          <v-col>
+                            <track-list
+                              v-if="appearance.Type === 'MusicAlbum'"
+                              :item="appearance"
+                            />
+                          </v-col>
+                        </v-row>
                       </div>
                     </v-col>
                   </v-row>
-                </v-container>
-              </v-tab-item>
-            </v-tabs-items>
-          </v-col>
-        </v-row>
-      </v-col>
-      <v-col cols="3">
-        <related-items :item="item" vertical>
-          {{ $t('moreLikeArtist', { artist: item.Name }) }}
-        </related-items>
-      </v-col>
-    </v-row>
-  </v-container>
+                </v-col>
+              </v-row>
+            </v-tab-item>
+            <v-tab-item :key="1">
+              <v-container>
+                <v-row>
+                  <v-col>
+                    <v-img
+                      cover
+                      aspect-ratio="1.7778"
+                      :src="getImageUrl(item.Id, 'Backdrop')"
+                    />
+                    <div v-if="item.Overview">
+                      <h2 class="text-h6 mt-2">
+                        <span>{{ $t('biography') }}</span>
+                      </h2>
+                      <v-col cols="12" sm="9" class="pl-0 pr-0">
+                        <!-- eslint-disable-next-line vue/no-v-html -->
+                        <p class="item-overview" v-html="overview" />
```

Comment:
```suggestion
                        <p class="item-overview" v-text="overview" />
```

Why not ``v-text``?

Reply:
Because some providers have markup or paragraphs in them, and these don't work without this.

Reply:
Imo we should just accept raw text from overviews, but that needs discussion and it's for another PR.